examples/first/main.go

```diff
@@ -16,30 +16,26 @@ import (
 	"github.com/beatlabs/patron/log"
 )
 
+const (
+	name    = "first"
+	version = "1.0.0"
+)
+
 func init() {
 	err := os.Setenv("PATRON_LOG_LEVEL", "debug")
 	if err != nil {
-		fmt.Printf("failed to set log level env var: %v", err)
-		os.Exit(1)
+		log.Fatalf("failed to set log level env var: %v", err)
```

Comment:
That's a good point. I wanted to simplify it due to `log.Fatalf` from stdlib does the same.
As for this case, it's not panicking because it uses `nilLogger` implementation that does nothing ( https://github.com/beatlabs/patron/blob/master/log/log.go#L51 )

Do we really need structure logs at this step? We could have something like fallbackLogger that uses the stdlib log package instead of `nilLogger`...